__init__.py

```
@@ -3,6 +3,43 @@
+    url = 'https://portaldaspalabras.gal/lexico/palabra-do-dia'
+    now = now_local()
+    data = {
+        'orde': 'data',
+        'comeza': '',
+        'palabra': '',
+        'data-do': f'{now.year}-{now.month}-{now.day}',
+        'data-ao': f'{now.year}-{now.month}-{now.day}',
+        'paged': ''
+    }
+
+    for i in range(3):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)